### PR TITLE
Results rules questions pages

### PR DESCRIPTION
**Pull request by varnita619**
- added questions  and options from firebase
- added functionality to select options
- added results page
 
 - image

- image

- image

### CODE REVIEW

File: src/Context/quizContext.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+const QuizContextProvider = ({ children }: { children: React.ReactNode }) => {
+  const [quizState, quizDispatch] = useReducer(quizReducer, initialState);
+  const [categoryQuiz, setCategoryQuiz] = useState("");
+
+  //Total score of the Specific user
+  // const totalScoreOfUser = data.reduce((sum: number, currentValue: any ) => sum + currentValue.score, 0)
+
+  // Getting categories from firebase
+
```
Comment (anuj-kumary) [inline src/Context/quizContext.tsx]: Remove unwanted comments

**Review by anuj-kumary — APPROVED** [review]
Clean code and Functionality is working as expected
Good to merge

Suggestion:-
- On your category page card images are not showing
- In your next button remove outline

File: src/Context/quizContext.tsx (hunk 1 above)
Comment (varnita619) [reply src/Context/quizContext.tsx]: Done

**Comment by varnita619** [comment]
> Clean code and Functionality is working as expected Good to merge
> 
> Suggestion:-
> 
> * On your category page card images are not showing
> * In your next button remove outline

**Comment by varnita619** [comment]
> Clean code and Functionality is working as expected Good to merge
> 
> Suggestion:-
> 
> * On your category page card images are not showing
> * In your next button remove outline

Done

**Review by pritam-kr — COMMENTED** [review]
Hi, I reviewed all pages, and all functionality is working nicely.
 
- Clean code
- good folder structure

**Review by pritam-kr — APPROVED** [review]

**Pull request merged**